packages/pn-landing-webapp/pages/informativa-privacy.tsx

```diff
@@ -2,13 +2,10 @@ import { useEffect } from "react";
 import type { NextPage } from "next";
 import Head from "next/head";
 
-// import { Typography, Grid, Link, Container } from "@mui/material";
-
-// const FAVICON_PATH = `${process.env.NEXT_PUBLIC_ASSETS_URL}/favicon.svg`;
-
 declare const OneTrust: any;
 
 const PrivacyPage: NextPage = () => {
+
```

Comment:
@AndreaCimini90 is it possible to move the privacy note link to a constant or env variable? so that it's easier to maintain

Reply:
Done